packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -124,13 +130,25 @@ export class Dropdown {
   private async registerListener(element: string | HTMLElement) {
     this.triggerElement = await this.resolveElement(element);
     if (this.triggerElement) {
-      this.triggerElement.addEventListener('click', this.openBind);
+      if (Array.isArray(this.triggerEvent)) {
+        this.triggerEvent.forEach((triggerEvent) => {
+          this.triggerElement.addEventListener(triggerEvent, this.openBind);
+        });
+      } else {
+        this.triggerElement.addEventListener(this.triggerEvent, this.openBind);
+      }
     }
   }
 
   private async unregisterListener(element: string | HTMLElement) {
     const trigger = await this.resolveElement(element);
-    trigger.removeEventListener('click', this.openBind);
+    if (Array.isArray(this.triggerEvent)) {
```

Comment:
Will this lead to an stale listener in the following usecase:

1. Load component with `triggerEvent = ['click', 'hover']`
2. Change trigger event during runtime to `triggerEvent = ['click']`

==> hover listener will not get removed